blog/content/post/2022-05-04-fobnail_provisioning.md

```diff
@@ -0,0 +1,292 @@
+---
+title: Fobnail Token - Fobnail provisioning
+abstract: 'This phase is about provisioning Fobnail Token itself. The closing
+          point of that process is creating a certificate for Token that can be
+          used later during attestation'
+cover: /covers/usb_token.png
+author: krystian.hebel
+layout: post
+published: true
+date: 2022-05-04
+archives: "2022"
+
+tags:
+  - fobnail
+  - tpm
+  - attestation
+categories:
+  - Security
+
+---
+
+# About the Fobnail Token project
+
+The Fobnail Token is a project that aims to provide a reference architecture for
```

Comment:
I copied it from previous posts, so it will have to be fixed there as well. It is also described this way at https://fobnail.3mdeb.com/

Reply:
@krystian-hebel Hard to say. Maybe someone else should take a look at that.

Reply:
@krystian-hebel Fobnail Token is just reference hardware essentially nRF840-Dongle from Nordic running our Fobnail Firmware. Goal of Fobnail Token is present Fobnail Project deliverables. Goals of Fobnail Project is to provide `reference architecture for building offline integrity measurement verifiers` in this case it is USB, but in other case it can be Bluetooth or NFC like in ideas from [Gabriel](https://gabriel.kerneis.info/2021/11/local_attestation/) - this concepts were also discussed during vPubs. Does it make sense?

Reply:
Makes sense. I should stop copying introductions without reading them first, I guess. I'll change it in previous posts too.

Reply:
@pietrushnic is it better now?

Reply:
yes, approved

Reply:
approved - so I guess we can resolve